vmdb/app/models/configuration_organization.rb

```diff
@@ -1,6 +1,11 @@
 class ConfigurationOrganization < ActiveRecord::Base
   belongs_to :provisioning_manager
   belongs_to :parent, :class_name => 'ConfigurationOrganization'
+  has_and_belongs_to_many :configuration_profiles
 
   alias_attribute :display_name, :title
+
+  def ancestry
```

Comment:
ASIDE: Seems we should have just used their value and translated all of them across and used the ancestry gem ourselves.